central/debug/service/service_test.go

```diff
@@ -168,3 +177,64 @@ func (s *DebugServiceTestSuite) TestGetConfig() {
 	s.NoError(err)
 	s.Equal(expectedConfig, actualConfig)
 }
+
+func (s *DebugServiceTestSuite) TestGetBundle() {
+	stubTime := time.Date(2023, 03, 14, 0, 0, 0, 0, time.UTC)
+	now = func() time.Time {
+		return stubTime
+	}
+
+	w := mockResponseWriter{
+		code: http.StatusOK,
+		data: bytes.NewBuffer(nil),
+	}
+	testutils.SetVersion(s.T(), testutils.GetExampleVersion(s.T()))
+	db := mocks.NewMockDB(s.mockCtrl)
+	pgxRows := pgxpoolmock.NewRows([]string{"server_version"}).AddRow("15.1").ToPgxRows()
+	pgxRows.Next() // This is a workaround for https://github.com/pashagolub/pgxmock/issues/133
+	db.EXPECT().QueryRow(gomock.Any(), "SHOW server_version;").Return(pgxRows)
+	globaldb.GetPostgresTest(s.T(), db)
+
+	s.configMock.EXPECT().GetConfig(gomock.Any()).Return(&storage.Config{}, nil)
+	s.service.writeZippedDebugDump(context.Background(), &w, "debug.zip", debugDumpOptions{
+		logs:              0,
+		telemetryMode:     0,
+		withCPUProfile:    false,
+		withLogImbue:      false,
+		withAccessControl: false,
+		withNotifiers:     false,
+		withCentral:       false,
+		clusters:          nil,
+		since:             time.Now(),
+	})
+
+	s.Equal(http.StatusOK, w.code)
+
+	body, err := io.ReadAll(w.data)
+	s.Require().NoError(err)
+
+	zipReader, err := zip.NewReader(bytes.NewReader(body), int64(len(body)))
+	s.Require().NoError(err)
+
+	for _, zipFile := range zipReader.File {
+		s.T().Log("Reading file:", zipFile.Name)
+		s.Assert().Equal(stubTime, zipFile.Modified.UTC())
+	}
+}
+
+type mockResponseWriter struct {
```

Comment:
An idea for small code improvement(we don't have to do it within this PR): there are currently 3 `mockResponseWriter` implementations in our code, perhaps we should unify them

Reply:
https://github.com/stackrox/stackrox/pull/5727